test/manipulation.test.js

```diff
@@ -1934,10 +2058,15 @@ describe('manipulation', function() {
 
     it('should only update instances that satisfy the where condition',
         function(done) {
-          Person.update({name: 'Brett Boe'}, {name: 'Harry Hoe'}, function(err,
+          var filter = (db.connector.name === 'cassandra') ? {id: idBrett} : {name: 'Brett Boe'};
```

Comment:
used connectorCapabilities.deleteWithOtherThanId and connectorCapabilities.reportDeletedCount
it seems the last check is still useful.